lib/library.rb

```diff
@@ -0,0 +1,37 @@
+require 'date'
+
+class Library
+  STANDARD_VALIDITY_MONTHS = 1
+
+  attr_accessor :collection, :exp_date, :shelf
```

Comment:
Is `exp_date` relevant to the entire Library or a Book? 

What is the differance between `shelf` and `collection`?